package.json

```diff
@@ -19,7 +19,23 @@
     "react-helmet": "^6.1.0"
   },
   "devDependencies": {
-    "prettier": "2.0.5"
+    "eslint": "^7.4.0",
+    "eslint-config-prettier": "^6.11.0",
```

Comment:
We wouldn't want random package-lock.json updates. So, please remove all the ^ and just have normal version number

Reply:
fixed.